stetho/src/main/java/com/facebook/stetho/Stetho.java

```diff
@@ -105,7 +106,7 @@ public static InspectorModulesProvider defaultInspectorModulesProvider(final Con
         modules.add(new Console());
         modules.add(new CSS());
         modules.add(new Debugger());
-        modules.add(new DOM());
+        modules.add(new DOM(new UIDOMProvider(context)));
```

Comment:
This should be internalized to `DOM`.  Just accept a `Context`.  This is a public API for Stetho and folks can subclass `DOM`.